api/src/wfl/service/gcs.clj

```diff
@@ -271,3 +271,13 @@
            :headers      (once/get-auth-header)
            :body         (json/write-str metadata :escape-slash false)}
           http/request :body (json/read-str :key-fn keyword)))))
+
+(defn userinfo
```

Comment:
Maybe split this into two functions, `who-made` taking a request, and `userinfo` taking an authorisation header?

Reply:
Won't do - I think it's unnecessary on reflection.